src/time_series_metadata_store.jl

```diff
@@ -1140,7 +1172,7 @@ function _create_row(
         ts_category,
         string(get_initial_timestamp(metadata)),
         Dates.Millisecond(get_resolution(metadata)).value,
-        Dates.Millisecond(get_horizon(metadata)),
+        Dates.Millisecond(get_horizon(metadata)).value,
```

Comment:
I should say that @josephmckinsey found this bug after debugging an error from @hannahchubin. I was surprised to learn
1. When SQLite.jl tries to write a value to SQLite, it will be direct pass thru if there is an obvious mapping (integer to integer). If a mapping doesn't exist, it will call `Serialization.serialize(value)` and store that as a BLOB in SQLite. The documentation does not make this obvious. The comments in the code make it clear that this may not work across Julia versions. I would have expected an error and be forced to opt-in to this behavior.
2. SQLite will happily store the BLOB in a column that you've defined as an integer.

Am I questioning past decisions? Maybe. I'm certainly not happy about this. Thanks @josephmckinsey.